forgerock-auth/src/main/java/org/forgerock/android/auth/Config.java

```diff
@@ -107,15 +108,15 @@ public void setSharedPreferences(SharedPreferences sharedPreferences) {
         this.sharedPreferences = sharedPreferences;
     }
 
-    // We need to remove this in future, because the connected tests are depends on this. but the unit tests/integration tests are not depend on this.
-    @Deprecated
```

Comment:
This is a breaking change, right? We need to publish a major release and have a note in the changelog about this potential breaking change... Let's bring this to the team because there might be other things that I'm forgetting...

Reply:
Developer should not call this interface, consider this an internal class. Most of the method under Config are more for testing purpose, except some special use cases.